feature/interests/src/test/kotlin/com/google/samples/apps/nowinandroid/interests/InterestsViewModelTest.kt

```diff
@@ -55,7 +54,8 @@ class InterestsViewModelTest {
     @Before
     fun setup() {
         viewModel = InterestsViewModel(
-            savedStateHandle = SavedStateHandle(mapOf(TOPIC_ID_ARG to testInputTopics[0].topic.id)),
+            // TODO: Figure out how to supply the correct dependency: InterestsDestination(topicId = testInputTopics[0].topic.id)
```

Comment:
This is ViewModel Unit test, so that I don't think here is the place to consider Navigation Args.
Manual insert `SavedStateHandle` looks fine to me.

Reply:
As mentioned on the comment for `TopicViewModelTest` this should be addressed in a new API.